build.gradle

```diff
@@ -247,3 +267,89 @@ if (project.hasProperty('nodeInstall')) {
 compileJava.dependsOn processResources
 processResources.dependsOn cleanResources,bootBuildInfo
 bootBuildInfo.mustRunAfter cleanResources
+
+idea.module.downloadSources = true
+
+task generateOpenApiSpec(dependsOn: 'classes', group: 'Swagger',
```

Comment:
It would be nice to add the open API spec to the docs folder of the release, or add it to the release files on github (draft release -> attach files)